### PR TITLE
[Torii] Debug output of all indexed Events

### PR DESCRIPTION
**Pull request by dubzn**
## Description
Display through `debug!()` the events that are not caught by the defined processors. The `keys` and `data` of the event are shown in hexadecimal values.

Closes: #1421

[Preview video](https://github.com/dojoengine/dojo/assets/58611754/e566f93b-c1ab-4229-85ca-7402e231d778
)

### CODE REVIEW

**Comment by tarrencev** [comment]
Hm I feel like `trace!` might be more appropriate here

File: crates/torii/core/src/engine.rs [hunk 1]
```diff
@@ -253,6 +253,10 @@ impl<'db, P: Provider + Sync> Engine<'db, P> {
                 processor
                     .process(&self.world, self.db, block, invoke_receipt, event_id, event)
                     .await?;
+            } else {
+                let keys: Vec<String> = event.keys.iter().map(|k| format!("{:#x}", k)).collect();
+                let data: Vec<String> = event.data.iter().map(|d| format!("{:#x}", d)).collect();
+                trace!("Event {{ keys: {:?}, data: {:?} }}", keys, data);
```
Comment (tarrencev) [inline crates/torii/core/src/engine.rs:259]: Can we use structured logging here?
```suggestion
                trace!("unprocessed event", keys = ?keys, data = ?data);
```

Comment (tarrencev) [reply crates/torii/core/src/engine.rs:259]: We should also start namespacing our loggers but that can be a future task

Comment (tarrencev) [reply crates/torii/core/src/engine.rs:259]: @dubzn let's merge this once you make the structured logging change

Comment (dubzn) [reply crates/torii/core/src/engine.rs]: Hey Tarrence, please check the changes and let me know wdyt. 
Sorry for the delay

**Pull request merged**